.changeset/funny-planes-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@openfn/language-satusehat': minor
+---
+
+Add temporary error logs
```

Comment:
I don't really like call these temporary here. I hope they are but still.

Can we just say "Added extra logging around errors" ?